VirtoCommerce.Platform.Web/Scripts/common/directives/dateTime-picker.js

```diff
@@ -0,0 +1,687 @@
+﻿angular.module('platformWebApp')
+    .constant('uiDatetimePickerConfig',
+        {
+            dateFormat: 'yyyy-MM-dd HH:mm',
+            defaultTime: '00:00:00',
+            html5Types: {
+                date: 'yyyy-MM-dd',
+                'datetime-local': 'yyyy-MM-ddTHH:mm:ss.sss',
+                'month': 'yyyy-MM'
+            },
+            initialPicker: 'date',
+            reOpenDefault: false,
+            enableDate: true,
+            enableTime: true,
+            buttonBar: {
+                show: true,
+                now: {
+                    show: true,
+                    text: 'Now'
```

Comment:
Remove and use location resources instead

Reply:
This text is not used for localization

Reply:
I've not completed review, just saved it to prevent loss of information. May be you need nothing to change. But if it is not used to display labels in UI, why need it at all?